usr/include/slotmgr.h

```diff
@@ -59,6 +66,16 @@
 #define NUMBER_PROCESSES_ALLOWED  1000
 #define NUMBER_ADMINS_ALLOWED     1000
 
+#if defined(_AIX)
+/*
+  AIX does not define NAME_MAX and instead requires this
+  to be determined at runtime. Use the equivalent defined for
+  BSD-compatibility, which is MAXNAMLEN - also 255
```

Comment:
you mention `MAXNAMLEN` here but use `_D_NAME_MAX` ?

Reply:
`MAXNAMLEN` is the BSD definition, although other platforms have also redefined it to be equal to their limits. On AIX, the value `_D_NAME_MAX` is defined, which is set to 255 for compatibility with BSD systems.

Reply:
OK, but I would then also mention `_D_NAME_MAX` in the comment, and not `MAXNAMLEN` .